apps/data_handler/db/models/zklend_events.py

```diff
@@ -39,3 +39,80 @@ class LiquidationEventData(EventBaseModel):
     debt_face_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
     collateral_token: Mapped[str] = mapped_column(String, nullable=False)
     collateral_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+
+
+class RepaymentEventData(EventBaseModel):
+    """
+    Database model for Repayment event, inheriting from EventBaseModel.
+
+    This model stores details of a repayment event, including the addresses involved
+    and the amounts in raw and face formats.
+    """
+
+    __tablename__ = "repayment_event_data"
+
+    repayer: Mapped[str] = mapped_column(String, nullable=False)
+    beneficiary: Mapped[str] = mapped_column(String, nullable=False)
+    token: Mapped[str] = mapped_column(String, nullable=False)
+    raw_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+    face_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+
+
+class DepositEventData(EventBaseModel):
+    """
+    Database model for Deposit event, inheriting from EventBaseModel.
+
+    This model stores details of a deposit event, including the user address,
+    token, and the face amount of the deposit.
+    """
+
+    __tablename__ = "deposit_event_data"
+
+    user: Mapped[str] = mapped_column(String, nullable=False)
+    token: Mapped[str] = mapped_column(String, nullable=False)
+    face_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+
+
+class BorrowingEventData(EventBaseModel):
+    """
+    Database model for Borrowing event, inheriting from EventBaseModel.
+
+    This model stores details of a borrowing event, including the user address,
+    token, and amounts in raw and face formats.
+    """
+
+    __tablename__ = "borrowing_event_data"
+
+    user: Mapped[str] = mapped_column(String, nullable=False)
+    token: Mapped[str] = mapped_column(String, nullable=False)
+    raw_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+    face_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+
+
+class WithdrawalEventData(EventBaseModel):
+    """
+    Database model for Withdrawal event, inheriting from EventBaseModel.
+
+    This model stores details of a withdrawal event, including the user address,
+    token, and the amount withdrawn.
+    """
+
+    __tablename__ = "withdrawal_event_data"
+
+    user: Mapped[str] = mapped_column(String, nullable=False)
+    amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+    token: Mapped[str] = mapped_column(String, nullable=False)
+
+
+class CollateralEnabledDisabledEventData(EventBaseModel):
+    """
+    Database model for CollateralEnabled/Disabled event, inheriting from EventBaseModel.
+
+    This model stores details of a collateral enabled/disabled event, including
+    the user address and the token involved.
+    """
+
+    __tablename__ = "collateral_enabled_disabled_event_data"
```

Comment:
we don't need to have here `_data`. please remove it
`collateral_enabled_disabled_event` will be okay